sisyphus/delayed_ops.py

```diff
@@ -142,6 +145,24 @@ def get(self):
         return try_get(self.a)[try_get(self.b)]
 
 
+class DelayedCall(DelayedBase):
+    def __init__(
+        self,
+        func: Union[DelayedBase, Callable],
+        args: Tuple[Union[DelayedBase, Any], ...],
+        kwargs: Dict[str, Union[DelayedBase, Any]],
+    ):
+        self.func = func
+        self.args = args
+        self.kwargs = kwargs
+
+    def get(self):
+        func = try_get(self.func)
+        args = [try_get(arg) for arg in try_get(self.args)]
+        kwargs = {key: try_get(value) for key, value in try_get(self.kwargs).items()}
```

Comment:
why the outer `try_get`?

```suggestion
        args = [try_get(arg) for arg in self.args]
        kwargs = {key: try_get(value) for key, value in self.kwargs.items()}
```
based on your typing info this should be valid.

Reply:
See other comment, I want to allow both delayed arg lists and kwarg dicts, and not just allow their contents to be delayed, but their structure fixed.